### ISSUE TITLE
Coop Players can grab props when respawning for few milliseconds(gamebug)

### ISSUE THREAD

**Issue by NoName50** [p2sr/SourceAutoRecord#211]
When you are respawning,you can grab things around you for few milliseconds
By doing this,you can take the prop to the player spawn point
But if you grab the prop at incorrect time,it(the fake one) will stick to your hand forever untill you change the map(you can shoot in this period)
(the actual prop becomes invisible and it's still at the original position)

**Comment by ThatFridgeFella** [comment]
I don't think that would be an issue specifically related to SAR, unless you only had this when testing a command in SAR.

**Comment by ThisAMJ** [comment]
related to `C_Portal_Player::ManageHeldObject`
